src/components/dialogs/create-case-dialog.js

```diff
@@ -84,8 +84,14 @@ export function CreateCaseDialog({ onClose, open }) {
             parentDirectoryUuid: activeDirectory,
         })
             .then()
```

Comment:
Shouldn't we also show this message if we create a study with a file having the wrong format ?

Reply:
i think we need to do the same for study for the same error

Reply:
ok